data/map planets.txt

```diff
@@ -277,6 +286,15 @@ planet Baianus
 	description `	Unlike Titan, however, Baianus is regularly wracked by both waves of heat and energy, and a stay of any length on the surface is likely to see waves of auroras accompanied by glows and sparks rippling across the terrain. Your environmental system strongly recommends remaining inside your ship whenever ion storms are active.`
 	government Uninhabited
 
+planet Bailey
+	attributes ccor mining
+	landscape land/lava3
+	description `Bailey is a dead, nearly tidally locked world scarcely larger and more habitable than Mercury. Its cool side has hardened into a mineral-rich crust, a rarity in this region of space. Despite its inhospitable conditions, Bailey's bountiful crust has brought several mining expeditions to the planet.`
+	description `	Like aphids sucking the sap from a plant, semi-mobile mining stations strip and refine the minerals, a method originally developed by the Confederated Councils. Since their fall, the pirates of the region have maintained the stations - although with considerably less organization than what was upheld by the central government.`
+	spaceport `Given the roving nature of the settlements, and the lack of a livable atmosphere, the spaceport and other major infrastructure are located on a repulsor-lift facility hovering a meter above Bailey's surface. Floating in place thanks to the lack of atmosphere, the planet slowly rotates beneath it while the facility is always on Bailey's dark side.`
```

Comment:
I'm not sure what the lack of atmosphere has to do with allowing the facility to levitate; gravity wells allow repulsors to operate. Taking this part out makes the description a little small, so maybe you could say that the facility is declared a neutral zone by the miners to mitigate disputes.

```suggestion
	spaceport `Given the roving nature of the settlements, and the lack of a livable atmosphere, the spaceport and other major infrastructure are located on a repulsor-lift facility hovering a meter above the surface of the dark side of the planet. Because of its importance to Bailey's economy, the facility has been declared neutral ground by the pirates and cartels that operate here; whenever disputes arise, the spaceport is the place where they are settled.`
```

Reply:
The lack of atmosphere is necessary, because otherwise you would encounter air resistance. The atmospheric drag would prevent the facility from remaining locked in place as the planet rotates.

Reply:
Ok.